### DISCUSSION TITLE
Cant mask with v-text-field vuetify 3

### DISCUSSION THREAD

**Discussion by inNetto** [beholdr/maska#142] [Q&A] [answered]
Im using Nuxt v3, Vuetify v3 and maska 2.1.7.

Maska not working with vuetify my custom component:

`<v-text-field
            v-model="cpf"
            v-maska
            data-maska="###.###.###.##"
            label="Cpf"
            :error-messages="errors.cpf"
          />`

But work fine with native component <input>

Im importing globally using plugin.

```
import { vMaska } from 'maska'

export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.directive('maska', vMaska)
})
```
Changing with this code, same problem.

```
const bindedObjedct = reactive({
      masked: '',
      unmasked: '',
      completed: false
})

<v-text-field
      v-model="cpf"
      v-maska="bindedObjedct"
      data-maska="#-#"
 />
```

The mask is applied with native input and i got the unmasked value either. But, in Vuetify component it doesnt apply the mask

**Comment by beholdr** [comment, marked as answer]
You can see vuetify example in this [thread](https://github.com/beholdr/maska/discussions/139).

**Reply by inNetto** [reply]
```
const maskaOptions = {
  mask: '+1 (###) ###-##-##',
  onMaska: (detail: MaskaDetail) => console.log(detail.unmasked)
}

<v-text-field
        v-model="cpf"
        v-maska:[maskaOptions]
        label="Cpf"
        :error-messages="errors.cpf"
/>


```
Worked. Thanks
